packages/ketcher-react/src/script/editor/Editor.ts

```diff
@@ -616,17 +616,18 @@ function updateLastCursorPosition(editor: Editor, event) {
 function domEventSetup(editor: Editor, clientArea) {
   // TODO: addEventListener('resize', ...);
   ;[
-    'click',
-    'dblclick',
-    'mousedown',
-    'mousemove',
-    'mouseup',
-    'mouseleave',
-    'mouseover'
-  ].forEach((eventName) => {
+    { target: clientArea, eventName: 'click' },
+    { target: clientArea, eventName: 'dblclick' },
+    { target: clientArea, eventName: 'mousedown' },
+    { target: document, eventName: 'mousemove' },
+    { target: document, eventName: 'mouseup' },
+    { target: document, eventName: 'mouseleave' },
```

Comment:
Hi 😃 seems it causes a little bug: pasted structure is here to stay when the mouse is moving outside the canvas.

**Steps to Reproduce**
1. Cut or copy a structure
2. `Ctrl+v` to paste it on the canvas
3. Move the mouse to the toolbar

**Actual behavior**
The pasted structure is still here
![ketcher-paste-bug-07](https://user-images.githubusercontent.com/27288153/213630048-087599c8-dba3-4c17-bdd1-d08105e4b6de.gif)


**Expected behavior**
The pasted structure disappears. The following GIF is created on v2.7.0 RC test remote version.
![ketcher-paste-bug-08](https://user-images.githubusercontent.com/27288153/213630486-9d3a4134-bbf2-4d0c-b081-340e4b27e6dd.gif)

Reply:
Do we have explicit requirements to remove it when over a toolbar?
Why should we use controversial logic for different tools? We do not break selection when going over toolbars, why should we break copy-paste action?

Reply:
You are right. Actually, sometimes I'm a bit confused about some controversial logic. Thanks for your fast reply :)

Reply:
And please check out this one. The hand tool is still following the cursor when moving outside the canvas.

![ketcher-paste-bug-09](https://user-images.githubusercontent.com/27288153/213639910-ad1dc04d-c0d6-4023-8dec-5a0d83efa34e.gif)



I'm not sure if it's caused by this line of code, but I guess it's because the `mouseleave` event is not triggered. The following code in `ketcher-react/src/script/ui/views/components/StructEditor/StructEditor.jsx` may help.

```jsx
case 'leave':
  this.editorRef.current.classList.remove(classes.enableCursor)
  this.setState({
    enableCursor: false
  })
  break
```

Reply:
Yeah, that does look weird. Thanks for noticing.
I think it deserves its own issue on the board. I'll look into it later.